docs/concepts/abstractions/controllers/garbage-collection.md

```diff
@@ -24,9 +24,12 @@ points to the owning object.
 
 Sometimes, Kubernetes sets the value of `ownerReference` automatically. For
 example, when you create a ReplicaSet, Kubernetes automatically sets the
-`ownerReference` field of each Pod in the ReplicaSet. You can also specify
-relationships between owners and dependents by manually setting the
-`ownerReference` field.
+`ownerReference` field of each Pod in the ReplicaSet. In 1.6, Kubernetes
+automatically sets the value of `ownerReference` for objects created by
+ReplicationController, Replicaset, StatefulSet, DaemonSet, and Deployment.
```

Comment:
This list is up-to-date for 1.6. You may want to capitalize Set in ReplicaSet for consistency.